evaluator/evaluate.go

```diff
@@ -255,7 +257,8 @@ func loadPackages(tx *gorm.DB, systemID int, data vmaas.UpdatesV2Response) (map[
 	err := tx.Table("package").
 		Select("concat(pn.name, '-', package.evra) nevra, package.*, sp.*").
 		Joins("join package_name pn on package.name_id = pn.id").
-		Joins("left join system_package sp on sp.system_id = ? and sp.package_id = package.id", systemID).
+		// nolint: lll
+		Joins("left join system_package sp on sp.rh_account_id = ? AND  sp.system_id = ? AND sp.package_id = package.id", accountID, systemID).
```

Comment:
better split it, using "``" for string definition. A bit more spaces then needed.

Reply:
Fixed by moving to `WHERE`

Reply:
Same here - I'd prefer to fix it in this commit not next one, just for the sake of clarity. ;)